tests/acceptance/pre-suite/02_chocolatey_application_install.rb

```diff
@@ -0,0 +1,49 @@
+require 'master_manipulator'
+require 'chocolatey_helper'
+test_name 'MODULES-3043 - C97739 - Install Client on Virgin System'
+
+chocolatey_pp = <<MANIFEST
+  class {'chocolatey':
+    chocolatey_download_url => 'file:///C:/chocolatey.nupkg',
+    use_7zip                => false,
+  }
+  include chocolatey
```

Comment:
Getting closer. I think you mean to remove this line (the lines above are basically the same).